pandas/core/frame.py

```diff
@@ -715,25 +715,25 @@ def _repr_html_(self):
     @Substitution(shared_params=fmt.common_docstring, returns=fmt.return_docstring)
     def to_string(
         self,
-        buf=None,
-        columns=None,
-        col_space=None,
-        header=True,
-        index=True,
-        na_rep="NaN",
-        formatters=None,
-        float_format=None,
-        sparsify=None,
-        index_names=True,
-        justify=None,
-        max_rows=None,
-        min_rows=None,
-        max_cols=None,
-        show_dimensions=False,
-        decimal=".",
-        line_width=None,
-        max_colwidth=None,
-    ):
+        buf: Optional[FilePathOrBuffer[str]] = None,
+        columns: Optional[List[str]] = None,
```

Comment:
docstring..

```
columns : sequence, optional, default None
    The subset of columns to write. Writes all columns by default.
```

so this needs one needs to change. may need to change in DataFrameFormatter as well.

this _does_ work with Series..

```python
>>> import pandas as pd
>>> pd.__version__
'0.26.0.dev0+350.gad9fe5d7a'
>>> ser = pd.Series(list("A"))
>>> df = pd.DataFrame({'A': [1,2,3], 'B': [4,5,6]})
>>> df
   A  B
0  1  4
1  2  5
2  3  6
>>> print(df.to_string(columns=ser))
   A
0  1
1  2
2  3
>>>
```

Reply:
`typing.Sequence[str]` for this?

Reply:
i _think_ so. (import Sequence from typing)

Reply:
> (import Sequence from typing)

yeah, obviously :smile: